datafusion/core/src/physical_optimizer/projection_pushdown.rs

```diff
@@ -1255,7 +1255,7 @@ fn update_join_filter(
                     side: col_idx.side,
                 })
                 .collect(),
-            join_filter.schema().clone(),
+            Arc::<arrow_schema::Schema>::clone(join_filter.schema()),
```

Comment:
I think this can be written more concisely like
```suggestion
            Arc::clone(join_filter.schema()),
```